src/js/requestsFactory.js

```diff
@@ -0,0 +1,21 @@
+const errorHandler = (async() => await import('./errorHandler.js'))();
+
+export default class RequestsFactory{
+    static async createRequest(httpMethod, requestQuery, requestParameters){
+        switch(httpMethod.toLowerCase()){
+            case 'get':
+                try{
+                    let x = await fetch(requestQuery).then(x=>x.json());
+                    return x;
```

Comment:
incorrect work with async/await throughout the project

consider following use:
const result = await fetch(requestQuery)
let jsonResult = await result.json();

Reply:
give meaningful names for variables